homeassistant/components/qnap_qsw/sensor.py

```diff
@@ -140,12 +143,12 @@ class QswSensorEntityDescription(SensorEntityDescription, QswEntityDescription):
         subkey=QSD_TX_SPEED,
     ),
     QswSensorEntityDescription(
-        translation_key="uptime",
```

Comment:
Can we keep the old one around and deprecate that one properly? Reolink did a PR a few days back where it had to deprecate one, so maybe that's a good inspiration